### DISCUSSION TITLE
Error, WebSocket

### DISCUSSION THREAD

**Discussion by yasharyan0904** [Cyfrin/foundry-full-course-cu#1927] [Q&A]
![Capture_E](https://github.com/Cyfrin/foundry-full-course-cu/assets/144695005/8f18dc08-1a65-4552-b579-161f34ef42d5)

**Comment by yasharyan0904** [comment]
Can anyone help me out, I just opened my vs code and suddenly this error came. It was working fine the day before :(

**Comment by yasharyan0904** [comment]
Restarted, and updated VS CODE. Disabled and Enabled the extensions. Now, its working fine :)

**Comment by EngrPips** [comment]
Awesome, Glad you figured that out. Keep crushing it. Friend.
